apps/remote_control/lib/lexical/remote_control/search/indexer.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Lexical.RemoteControl.Search.Indexer do
+  alias Lexical.Project
+  alias Lexical.RemoteControl.Search.Indexer
+
+  def index(%Project{} = project) do
```

Comment:
What's the use case for this function? Won't all modules be available anyway after compilation?

Reply:
So no, nothing in the test directories, nor any .exs files will be available.

Reply:
That's true